src/cr/cube/cube.py

```diff
@@ -506,36 +503,35 @@ def _measures(self):
 
     @lazyproperty
     def _numeric_array_dimension(self):
-        """Column dimension object according to the mean subvariables."""
+        """Row dimension object according to the mean subvariables."""
```

Comment:
I think we need to use "rows dimension" wording, rather than "row dimension". Plural is an important part of how we  view the measures and everything...